MewsSystems/mews-flutter
# feat: [DX-1521] Update icons library

#### Summary

- added new icons:
  - `caret_filled_up`
  - `caret_filled_down`
- updated `All Icons` story

#### Testing steps

1. Open `All Icons` story
2. Check if the icons weren't messed up (for some reason sometimes `fluttericon.com` would generate the wrong `name: icon` pairs, so it needs to be double-checked).

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
